### PR TITLE
Merge CMake build system needed for DFTB+ build

### PR DESCRIPTION
**Pull request by awvwgk**
@dftbplus/administrators Any objections to bring the CMake build system to this repository and use it for the fetch script in DFTB+ to fix https://github.com/dftbplus/dftbplus/issues/875

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -13,15 +13,14 @@ split into two parts:
 * A library with the core functionality. This can be directly used by third
   party applications wishing to calculate dispersion with the DFT-D3
   approach.
-
+
```
Comment (bhourahine) [inline README.rst:16]:
```suggestion

```

File: README.rst [hunk 2]
```diff
@@ -47,8 +46,8 @@ When using the library or the dftd3 tool, please cite:
 
   S. Grimme, J. Antony, S. Ehrlich and H. Krieg
   J. Chem. Phys, 132 (2010), 154104.
-
-If BJ-damping is used
+
```
Comment (bhourahine) [inline README.rst:49]:
```suggestion

```

File: README.rst [hunk 3]
```diff
@@ -47,8 +46,8 @@ When using the library or the dftd3 tool, please cite:
 
   S. Grimme, J. Antony, S. Ehrlich and H. Krieg
   J. Chem. Phys, 132 (2010), 154104.
-
-If BJ-damping is used
+ 
+If BJ-damping is used
```
Comment (bhourahine) [inline README.rst:50]:
```suggestion
If BJ-damping is used
```

**Comment by akesandgren** [comment]
Also, please make sure to produce a new release of DFTB+ once this change has been made and the get_opt_externals has been updated. Otherwise no one will notice...

**Comment by aradi** [comment]
@akesandgren Thanks a lot for finding and reporting this. @awvwgkThanks for setting this up. There was no special reason, why it was in my personal fork instead in upstream/main (apart of my forgetfulness to make a PR of it). And yes, @bhourahine is absolutely right, we must fix `get_opt_externals` to fetch its stuff from the official repo.

**Pull request merged**